src/eel/eel/pressure/pressure_sensor.py

```diff
@@ -1,14 +1,14 @@
+from typing import Union, cast
 import serial
 import struct
 import time
-
 from rclpy.node import Node
-
+from .pressure_source import PressureSource
```

Comment:
The abstract base class is that to be able to support future different implementations of pressure sensor?

Reply:
Mostly added it for better type checking in the editor. Both the simulated sensor and the real one only have one important method, `get_current_depth` or whatever it was called. But since we say `self.sensor = simulation_sensor OR real_sensor`, the type of `self.sensor` is `Any` or `Unknown` or something like that. So the method `get_current_depth` doesn't exist. But we can say that they both inherit from an abstract class, which has that method. And then we get the correct return types and so on.